.github/workflows/chromatic.yml

```
@@ -17,7 +17,40 @@ jobs:
       - name: Kirby setup
         uses: ./.github/actions/kirby-setup
 
-      - name: Publish to Chromatic
+      - name: Derive appropriate SHAs for base and head for `nx affected` commands
+        uses: nrwl/nx-set-shas@v4
+        with: 
+          main-branch-name: develop
+
+      - name: Build storybook
+        run: npx nx affected -t build-storybook
+
+      - name: Check if storybook dist folders exist
+        id: check_storybook_folders
+        run: |
+          if [ -d ${{ github.workspace }}/dist/storybook/designsystem ]; then
+            echo "designsystem=true" >> $GITHUB_OUTPUT
+          fi
+          if [ -d ${{ github.workspace }}/dist/storybook/extensions/angular ]; then
+            echo "extensions_angular=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Publish Designsystem  to Chromatic
```

Not important: Two spaces between words

```suggestion
      - name: Publish Designsystem to Chromatic
```